xtask/src/ci/target_matrix.rs

```diff
@@ -0,0 +1,111 @@
+use clap::builder::BoolishValueParser;
+use clap::Parser;
+use serde::Serialize;
+
+use crate::util::{get_matrix, gha_output, gha_print, CiTarget, ImageTarget};
+
+pub(crate) fn run(message: String, author: String) -> Result<(), color_eyre::Report> {
+    let mut matrix: Vec<CiTarget> = get_matrix().clone();
+    if author == "bors[bot]" && message.starts_with("Try #") {
+        if let Some((_, args)) = message.split_once(": ") {
+            let app = TargetMatrixArgs::parse_from(args.split(' '));
+            if app.target.is_empty() {
+            } else {
+                matrix.retain(|m| {
+                    let matrix_target = m.to_image_target();
+                    let matrix_string = matrix_target.to_string();
+
+                    app.target
+                        .iter()
+                        .any(|t| t.parse::<ImageTarget>().unwrap() == matrix_target)
+                        || app
+                            .target
+                            .iter()
+                            .any(|t| matrix_string.contains(t.as_str()))
+                })
+            };
+            if let Some(std) = app.std {
+                matrix.retain(|m| m.std.unwrap_or_default() == std)
```

Comment:
Is this just so we can provide `bors try --std`, to try all targets supporting `std`? Same with everything else, obviously? Same with everything else including `runners`?

Reply:
yes, I don't see the need for it immediately but you never know.

Reply:
I think it's good then: future-proofing or unexpected needs is quite good. There might be a situation we only want targets that are `no-std`, and this would dramatically simplify the CI steps.